contrib/bool_plperl/meson.build

```diff
@@ -23,7 +23,24 @@ bool_plperl = shared_module('bool_plperl',
     'build_rpath': '@0@/CORE'.format(archlibexp),
   },
 )
-contrib_targets += bool_plperl
```

Comment:
Seems like you accidentally deleted this.

Reply:
it's adjusted/fixed on the last commit e4cb1230cae8b3c5c885c35aefb56294b505c0e0

Reply:
My fault, I wrote the wrong thing. I meant to write "Seems like you accidentally changed this line.".

Reply:
Not as far as I can see.

Reply:
Nevermind, fixup that completely restores contrib_targets even when `llvm.found() == false`
https://github.com/percona/postgres/commit/f26737a419783745059b95494dd72e51c309755b